packages/nx/src/plugins/js/project-graph/build-dependencies/target-project-locator.ts

```diff
@@ -31,21 +31,11 @@ type NpmResolutionCache = Map<string, string | null>;
  */
 const defaultNpmResolutionCache: NpmResolutionCache = new Map();
 
-const builtInModuleSet = new Set<string>([
-  ...builtinModules,
-  ...builtinModules.map((x) => `node:${x}`),
-  // These are missing in the builtinModules list
-  // See: https://github.com/nodejs/node/issues/42785
-  // TODO(v20): We should be safe to use `isBuiltin` function instead of keep the set here (https://nodejs.org/api/module.html#moduleisbuiltinmodulename)
-  'test',
-  'node:test',
-  'node:sea',
-  'node:sqlite',
-]);
+const experimentalNodeModules = new Set(['node:sqlite']);
```

Comment:
Is this necessary? If you run with `node --experimental-sqlite` then `isBuiltin` from `node:module` already includes `node:sqlite`.

Reply:
Sometimes Nx Console starts the daemon and probably won't run with `--experimental-sqlite` so this ensure that it's handled regardless of it's passed or not.